app/src/main/java/com/github/steroidteam/todolist/todo/TodoListCollection.java

```diff
@@ -0,0 +1,33 @@
+package com.github.steroidteam.todolist.todo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class TodoListCollection {
```

Comment:
Why creating this class, instead of using directly an `ArrayList<TodoList>`? Seems like we're basically wrapping the main methods of a normal `ArrayList`.